sdk-webapp/i18n/en.ts

```diff
@@ -686,14 +686,14 @@
     <name>set_as_default</name>
     <message>
         <source>set as default</source>
-        <translation>set as default</translation>
+        <translation>Set As Default</translation>
```

Comment:
Capitalizing "as" in cases like "Save As" or "Save Page As" is an exception to the rule, since there "as" is the last word of the title. Generally it should be written in lowercase (e.g. "Save as PDF" or "Set as Default"). This is at least how I've understood the Chicago manual of style and e.g. Apple and Microsoft seem to follow this practise in their applications.